website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,220 @@
+### Batch filtering
+
+Batch filtering supports the user making multiple selections across different parameters and only updating the results of the data set when the user interacts with a global "Apply" function. This method is useful when the data set is very large or when the filter parameters are complex.
+
+<video width="100%" controls loop>
+  <source
+    src="/assets/patterns/filter-patterns/batch-filtering.mp4"
+    type="video/mp4"
+  />
+</video>
+
+## Displaying selected filters
+
+Communicate that values corresponding with a filter parameter have been applied within the [Dropdown](/components/dropdown) by adding a `BadgeCount` within the filter. This can more easily communicate to the user what filters and values have been applied.
+
+**Communicating that two values within a parameter have been applied in the Dropdown**
+
+![Filter dropdown with count](/assets/patterns/filter-patterns/filter-bar-dropdown-count.png =661x*)
+
+This, when combined with the applied filters provides detailed snapshot of what specific filter values are applied to the data set and the relevant parameters that they correspond with.
+
+**Communicating that multiple filter parameters have been applied**
+
+![Filter dropdown with multiple counts](/assets/patterns/filter-patterns/filter-bar-dropdown-count-multiple.png =683x*)
+
+## Overflow
+
+Depending on the complexity of the data set, it may be necessary to account for the overflow of elements within the filter bar or in the applied tags.
+
+### Filter bar overflow
+
+If the number of filterable parameters exceeds the available space, consider prioritizing the most commonly used filters in the filter bar and moving less important or more complex filters to a [Flyout](/components/flyout) that can be triggered via an action in the filter bar.
+
+**Hiding additional filters behind an action; in this scenario a secondary button**
+
+![More filters trigger](/assets/patterns/filter-patterns/overflow-filter-bar.png =559x*)
+
+**Triggering a Flyout with additional filter parameters**
+
+![Flyout triggered by more filters](/assets/patterns/filter-patterns/overflow-filter-bar-flyout.png =559x*)
+
+!!! Dont
+
+Don't stack multiple filter bars on top of a data set. This can result in unnecessary complexity in the UI.
+
+![Multiple filter bar rows](/assets/patterns/filter-patterns/overflow-filter-bar-multiple-rows.png =559x*)
+
+!!!
+
+### Filter sidebar overflow
+
+Similar to the guidelines for the filter bar, an action to view more filters within a [Flyout](/components/flyout) can be triggered within the sidebar orientation.
+
+**Hiding additional filters behind an action in a sidebar orientation**
+
+![More filters trigger in a sidebar](/assets/patterns/filter-patterns/overflow-sidebar-more-filters.png =559x*)
+
+!!! Info
+
+Consider each one of these scenarios carefully as moving additional filters to an interstitial component like a Flyout adds extra steps in the filtering process that might not be necessary.
+
+!!!
```

Comment:
Can you help me understand why this wouldn't just scroll to reveal more filters?

Reply:
Updated this!